raiden/network/transport/matrix/rtc/utils.py

```diff
@@ -0,0 +1,10 @@
+import gevent
+import nest_asyncio
```

Comment:
We ran into a problem when using the `--console` argument because iPython is calling `asyncio.get_event_loop().run_until_complete()` while the event loop is already running causing a crash

Reply:
Then I'd say we should only use this when the user actually asks for the console.
That also means we can move it to the dev dependencies.